ksqldb-engine/src/main/java/io/confluent/ksql/services/SandboxConnectClient.java

```diff
@@ -33,18 +36,16 @@ private SandboxConnectClient() {
 
   }
 
-  public static ConnectClient createProxy() {
+  public static ConnectClient createProxy(final ConnectClient delegate) {
     return LimitedProxyBuilder.forClass(ConnectClient.class)
-        .swallow("create", methodParams(String.class, Map.class),
-            ConnectResponse.failure("sandbox", HttpStatus.SC_INTERNAL_SERVER_ERROR))
-        .swallow("describe", methodParams(String.class),
-            ConnectResponse.failure("sandbox", HttpStatus.SC_INTERNAL_SERVER_ERROR))
+        .forward("validate", methodParams(String.class, Map.class), delegate)
         .swallow("connectors", methodParams(),
             ConnectResponse.success(ImmutableList.of(), HttpStatus.SC_OK))
-        .swallow("status", methodParams(String.class),
-            ConnectResponse.failure("sandbox", HttpStatus.SC_INTERNAL_SERVER_ERROR))
-        .swallow("delete", methodParams(String.class),
-            ConnectResponse.success("sandbox", HttpStatus.SC_NO_CONTENT))
+        .swallow("create", methodParams(String.class, Map.class),
```

Comment:
Not sure what is the reason behind changing the existing ``create`` behavior?

Reply:
No, that's not enough. The datum type needs to match what's expected here, else this will throw: https://github.com/confluentinc/ksql/blob/7b3fb6b078a02c48ab4fe976ca836b8efe514a59/ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/ConnectExecutor.java#L75

Reply:
That said, in the latest version of this PR the create() command is not part of the validation at all so I've simply removed this line from the sandbox client.